pkg/apis/agones/v1/gameserver.go

```diff
@@ -192,7 +192,9 @@ type GameServerPort struct {
 	// When `Static` portPolicy is specified, `HostPort` is required, to specify the port that game clients will
 	// connect to
 	PortPolicy PortPolicy `json:"portPolicy,omitempty"`
-	// ContainerPort is the port that is being opened on the game server process
+	// ContainerName is the container on which to open the port. Defaults to the game server container.
+	ContainerName string `json:"containerName,omitempty"`
```

Comment:
Should this be `Container string` to match what we have at the top level for specifying the "game server container"? Seems like it should be more consistent.

Also, when you do the Alpha feature flagging - this should be made into a pointer, so it's easily omitted.